src/components/Menu.jsx

```diff
@@ -1,29 +1,70 @@
 import React, { useState, useEffect } from "react";
-import { Segment, List } from "semantic-ui-react";
+import { Segment, Grid, Menu, Container } from "semantic-ui-react";
 import MenuItem from "./MenuItem";
 import axios from "axios";
 
-const Menu = () => {
+const MenuPage = () => {
   const [menuItems, setMenuItems] = useState([]);
+  const [activeCategory, setActiveCategory] = useState("starters");
+
   useEffect(() => {
     axios.get("https://slowfood.heroku.com/api/products").then((response) => {
       setMenuItems(response.data.products);
     });
-  }, []);
+  }, [activeCategory]);
+
+  const itemsInCategories = menuItems.filter(
+    (item) => item.category === activeCategory
+  );
 
-  let menuList = menuItems.map((item) => {
-    return (
-      <List data-cy={`item-${item.id}`}>
-        <MenuItem item={item} />
-      </List>
-    );
+  let menuList = itemsInCategories.map((item) => {
+    return <MenuItem key={item.id} item={item}></MenuItem>;
```

Comment:
If `MenuItem` has no children, then you can simplify this a bit: 

```js
<MenuItem key={item.id} item={item} />
```

Reply:
We will remove the extra code snippet.